solidity/core/test/validator-manager/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { types, Validator } from '@abacus-network/utils';
+
+// Signs a checkpoint with the provided validators and returns
+// the signatures sorted by validator addresses in ascending order
+export async function signCheckpoint(
+  root: types.HexString,
+  index: number,
+  unsortedValidators: Validator[],
+): Promise<string[]> {
+  const validators = unsortedValidators.sort((a, b) => {
+    // Remove the checksums for accurate comparison
+    const aAddress = a.address.toLowerCase();
+    const bAddress = b.address.toLowerCase();
+
```

Comment:
nice
I wonder if ethers has something like this

Reply:
poking around I don't see anything :/